lib/utils.js

```diff
@@ -250,6 +250,15 @@ function isSignedBy(message, signature, address) {
     }
 }
 
+/**
+ * Checks whether it is expected revert contract exception.
+ * @param {Error} error - error object thrown by ethers contract funcion call
+ * @returns {Boolean} True if it is revert exception thrown by the contract
+ */
+function isRevertContractException(error) {
```

Comment:
Is this function really needed as an exported function from the SDK? Why should anyone outside the settlement implementations need to know about this type of error?

Reply:
You are right. For now, It is only used for the settlement operations. Perhaps I need to make it as a private function in the DriipSettlement and NullSettlement classes, but that could duplicate the code logic for that function. Another way is to create a `utils.js` inside the settlement folder to expose these types of functions.

Reply:
Having a local "utils" inside the settlement folder sounds like a solution for now. I cant see that anyone outside "settlements" is using that function at present.